probability.ipynb

```diff
@@ -74,12 +72,21 @@
       "text/html": [
        "<!DOCTYPE html PUBLIC \"-//W3C//DTD HTML 4.01//EN\"\n",
        "   \"http://www.w3.org/TR/html4/strict.dtd\">\n",
-       "\n",
+       "<!--\n",
+       "generated by Pygments <http://pygments.org>\n",
+       "Copyright 2006-2019 by the Pygments team.\n",
+       "Licensed under the BSD license, see LICENSE for details.\n",
+       "-->\n",
        "<html>\n",
        "<head>\n",
        "  <title></title>\n",
        "  <meta http-equiv=\"content-type\" content=\"text/html; charset=None\">\n",
        "  <style type=\"text/css\">\n",
+       "/*\n",
```

Comment:
Same here, and all the ones below. If we are not using Pygments, it'd be better to not have these comments here. A simple find-replace would do the job.

Reply:
Thanks! removed all the irrelevant lines and made a new commit. It should be all right now.